cli/flags.rs

```diff
@@ -2409,6 +2415,24 @@ fn inspect_arg_validate(val: String) -> Result<(), String> {
   }
 }
 
+fn watch_arg_parse(
+  flags: &mut Flags,
+  matches: &clap::ArgMatches,
+  allow_extra: bool,
+) {
+  flags.watch = match matches.values_of("watch") {
+    Some(f) => {
+      let extra_files = f.map(PathBuf::from).collect();
+      if allow_extra {
+        Some(extra_files)
+      } else {
+        Some(vec![])
```

Comment:
I thought maybe it could be nice to panic here or show some sort of error when external files are provided for any non `run` command. But I'm not sure of a good way to do this. Is a `panic!` actually acceptable, or is there a better way to handle invalid arguments that I'm not aware of?

Reply:
Panicking would be no good, I think it's fine to leave it as is and just ignore these files.